terraform/deployments/govuk-publishing-platform/app_static.tf

```
@@ -1,51 +1,98 @@
+locals {
+  static_defaults = {
+    cpu    = 512  # TODO parameterize this
+    memory = 1024 # TODO parameterize this
+
+    environment_variables = merge(
+      local.defaults.environment_variables,
+      {
+        GOVUK_APP_NAME                   = "static",
+        GOVUK_APP_ROOT                   = "/var/apps/static",
+        PLEK_SERVICE_ACCOUNT_MANAGER_URI = "",
+        REDIS_URL                        = local.defaults.redis_url,
+        RAILS_SERVE_STATIC_FILES         = "enabled",
+        RAILS_SERVE_STATIC_ASSETS        = "yes",
+      }
+    )
+
+    secrets_from_arns = local.defaults.secrets_from_arns
+  }
+}
+
 module "static" {
+  source = "../../modules/app"
+
   service_name                     = "static"
   mesh_name                        = aws_appmesh_mesh.govuk.id
   service_discovery_namespace_id   = aws_service_discovery_private_dns_namespace.govuk_publishing_platform.id
   service_discovery_namespace_name = aws_service_discovery_private_dns_namespace.govuk_publishing_platform.name
-  subnets                          = local.private_subnets
-  vpc_id                           = local.vpc_id
   cluster_id                       = aws_ecs_cluster.cluster.id
-  source                           = "../../modules/app"
+  vpc_id                           = local.vpc_id
+  subnets                          = local.private_subnets
   desired_count                    = var.static_desired_count
   extra_security_groups            = [local.govuk_management_access_security_group, aws_security_group.mesh_ecs_service.id]
+  environment_variables = merge(
+    local.static_defaults.environment_variables,
+    {
+      ASSET_HOST          = "https://static-ecs.${local.defaults.environment_variables.GOVUK_APP_DOMAIN_EXTERNAL}",
```

🤔  I wonder what this should actually be... The other apps use `local.defaults.asset_host`, which is currently set to `https://frontend.${var.external_app_domain}`.

If this needs to be `https://static-ecs...` then I think:

```suggestion
      ASSET_HOST          = "https://static-ecs.${var.external_app_domain}",
```

would be nicer.

---

I can do that. In `test` environment, ASSET_HOST for static is `https://www.test.publishing.service.gov.uk` and router is supposed to route it to the right app. Since we don't have router working properly, this is a temp fix. I can add a TODO